UefiPayloadPkg/UefiPayloadPkg.fdf

```diff
@@ -15,8 +15,12 @@ DEFINE FD_BLOCK_SIZE = 0x00001000
 !if $(TARGET) == "NOOPT"
 DEFINE FD_SIZE     = 0x00850000
 DEFINE NUM_BLOCKS  = 0x850
-!else
 
+!elseif $(USE_CBMEM_FOR_CONSOLE) == TRUE
+DEFINE FD_SIZE     = 0x00600000
+DEFINE NUM_BLOCKS  = 0x600
+

```

Comment:
Why need to change the FD size?

Reply:
There isn't enough space, the build will fail